UPGRADE-4.x.md

```diff
@@ -1,6 +1,13 @@
 UPGRADE 4.x
 ===========
 
+UPGRADE FROM 3.x to 3.x
```

Comment:
4.x to 4.x

Reply:
I will fix it in next PR, also merge today (before release): #1230